tests/e2e/UsingBluemix.md

```diff
@@ -0,0 +1,95 @@
+# Run E2E tests on Bluemix
+
+This setup helps you run e2e tests with your local codes against a k8s cluster on BLuemix.
+
+* [Step 0: Install and configure IBM Cloud Kubernetes Service CLI](#step-0-install-and-configure-ibm-cloud-kubernetes-service-cli)
+* [Step 1: Create and configure your kubernetes cluster](#step-1-create-and-configure-your-kubernetes-cluster)
+* [Step 2: Set up Registry](#step-2-set-up-registry)
+* [Step 3: Run the tests](#step-3-run-the-tests)
+
+## Step 0: Install and configure IBM Cloud Kubernetes Service CLI
+
+### 1. Download and Install IBM Cloud CLI [Full Instructions](https://console.bluemix.net/docs/cli/reference/bluemix_cli/download_cli.html#download_install):
```

Comment:
### 1. Download and Install IBM Cloud CLI

...

See [full Instructions](https://console.bluemix.net/docs/cli/reference/bluemix_cli/download_cli.html#download_install) for more details.